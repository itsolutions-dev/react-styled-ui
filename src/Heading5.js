import styledComponent, { css } from './styledComponent';
import { getTextColor, getColorWithAlpha } from './utils/';

const Heading5 = styledComponent('h5', css`
  font-size: ${props => props.theme.h5.fontSize};
  font-weight: ${props => props.theme.h5.fontWeight};
  line-height: ${props => props.theme.h5.lineHeight};
  color: ${props => getColorWithAlpha(getTextColor(props), 0.54)};
`);

export default Heading5;
