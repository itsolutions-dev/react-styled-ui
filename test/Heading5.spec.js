import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import 'styled-components-test-utils/lib/jest';
import Heading5 from '../src/Heading5';

describe('Heading5', () => {
  test('should render a Button', () => {
    const component = ReactTestRenderer.create(<Heading5 />);
    expect(component).toBeDefined();
  });

  test('should have a h5 tag', () => {
    expect(Heading5).toHaveComponent('h5');
  });
});
