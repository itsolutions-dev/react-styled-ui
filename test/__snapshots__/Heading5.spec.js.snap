// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading5 should match snapshot 1`] = `
.bhzmJR {
  font-size: 29px;
  font-weight: 900;
  line-height: 450;
  color: rgba(255, 255, 255, 0.54);
}

<h5
  className="sc-bdVaJa bhzmJR"
/>
`;
